cmd/importer/data.go

```diff
@@ -32,34 +31,30 @@ type datum struct {
 	timeValue   time.Time
 	remains     uint64
 	repeats     uint64
+	probability uint32
 	step        int64
 
 	init     bool
 	useRange bool
 }
 
 func newDatum() *datum {
-	return &datum{intValue: -1, step: 1, repeats: 1, remains: 1}
+	return &datum{step: 1, repeats: 1, remains: 1, probability: 100}
 }
 
-func (d *datum) setInitInt64Value(step int64, min int64, max int64) {
+func (d *datum) setInitInt64Value(min int64, max int64) {
 	d.Lock()
 	defer d.Unlock()
 
 	if d.init {
 		return
 	}
 
-	d.step = step
-
-	if min != -1 {
-		d.minIntValue = min
-		d.intValue = min
-	}
-
-	if min < max {
-		d.maxIntValue = max
-		d.useRange = true
+	d.minIntValue = min
+	d.maxIntValue = max
+	d.useRange = true
+	if d.step < 0 {
```

Comment:
should we specially handle 0 step?

Reply:
If step is 0, we can use `range` or `set` rule, instead of `incremental`?

Reply:
@lamxTyler How do you think?

Reply:
0 step also makes senses, for example, we only want constant data.

Reply:
Yes, but if we want constant data, we don't need to use `incremental`, a better way is using `set`?

Reply:
I prefer to do this optimization in the future, for now, let's make it work.